database/migrations/model/Version20220418172350.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Database\Migrations\Model;
+
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema as Schema;
+use LaravelDoctrine\Migrations\Schema\Builder;
+use LaravelDoctrine\Migrations\Schema\Table;
+
+class Version20220418172350 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema):void
+    {
+        $builder = new Builder($schema);
+        if(!$schema->hasTable("Summit_RegistrationCompanies")) {
+            $builder->create('Summit_RegistrationCompanies', function (Table $table) {
+                $table->integer("ID", true, false);
+                $table->primary("ID");
+                $table->integer("SummitID", false, false)->setNotnull(false);
+                $table->index("SummitID", "SummitID");
+                $table->foreign("Summit", "SummitID", "ID", ["onDelete" => "CASCADE"]);
```

Comment:
please assign a name to the FK as last param
like 'FK_RegistrationCompanies_Summit'